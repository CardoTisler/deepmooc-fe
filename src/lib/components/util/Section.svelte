<script lang="ts">
	let style = '';
	export { style as class };
</script>

<div class="border-2 border-gray-200 rounded-xl bg-white w-full {style}">
	<slot />
</div>
