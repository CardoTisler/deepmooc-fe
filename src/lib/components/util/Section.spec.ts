import { render } from '@testing-library/svelte';
import Section from './Section.svelte';

describe('Section.svelte', () => {
	it('should render component', () => {
		const { container } = render(Section);
		expect(container).toMatchSnapshot();
	});
});
