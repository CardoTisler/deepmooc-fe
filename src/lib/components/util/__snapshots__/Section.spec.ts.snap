// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Section.svelte > should render component 1`] = `
<body>
  <div>
    <div
      class="border-2 border-gray-200 rounded-xl bg-white w-full dark:border-yellow-300  s-u_aBuH8kPm0b"
    />
    <!--&lt;Section&gt;-->
  </div>
</body>
`;
